gyp/platform-ios.gypi

```diff
@@ -30,6 +30,12 @@
         '../include/mbgl/ios/MGLMetricsLocationManager.h',
         '../platform/ios/MGLMetricsLocationManager.m',
         '../include/mbgl/ios/MGLTypes.h',
+        '../platform/ios/NSProcessInfo+MGLAdditions.h',
+        '../platform/ios/NSProcessInfo+MGLAdditions.m',
+        '../platform/ios/NSString+MGLAdditions.h',
+        '../platform/ios/NSString+MGLAdditions.m',
+        '../include/mbgl/ios/MGLMapView_IBAdditions.h',
+        '../platform/ios/MGLMapView_IBAdditions.m',
```

Comment:
Answered in https://github.com/mapbox/mapbox-gl-native/pull/1184/files#r27702538.